features/logout/impl/src/main/res/values-fr/translations.xml

```diff
@@ -3,6 +3,16 @@
   <string name="screen_signout_confirmation_dialog_content">"Êtes-vous sûr de vouloir vous déconnecter ?"</string>
   <string name="screen_signout_confirmation_dialog_title">"Se déconnecter"</string>
   <string name="screen_signout_in_progress_dialog_content">"Déconnexion…"</string>
+  <string name="screen_signout_key_backup_disabled_subtitle">"Vous êtes en train de vous déconnecter de votre dernière session. Si vous vous déconnectez maintenant, vous perdrez l’accès à l’historique de vos discussions chiffrées."</string>
+  <string name="screen_signout_key_backup_disabled_title">"Vous avez désactivé la sauvegarde"</string>
+  <string name="screen_signout_key_backup_offline_subtitle">"Vos clés étaient en cours de sauvegarde avant que vous ne perdiez le réseau. Il faudrait retrouver le réseau afin de terminer la sauvegarde avant de vous déconnecter."</string>
```

Comment:
```suggestion
  <string name="screen_signout_key_backup_offline_subtitle">"Vos clés étaient en cours de sauvegarde lorsque vous avez perdu la connexion au réseau. Il faudrait rétablir cette connexion afin de pouvoir terminer la sauvegarde avant de vous déconnecter."</string>
```

Reply:
Updated (note: this string is not used on Android, maybe used on iOS.)